lib/domain/exception/room/can_not_create_new_group_chat_exception.dart

```
@@ -0,0 +1,3 @@
+class CannotCreateNewGroupChatException implements Exception {
+  CannotCreateNewGroupChatException() : super();
```

how about the root cause `e`? Please don't swallow the error